dso-l1/src/main/java/com/tc/config/schema/setup/ClientConfigurationSetupManagerFactory.java

```diff
@@ -34,7 +34,7 @@
  */
 public class ClientConfigurationSetupManagerFactory {
 
-  private static final TCLogger consoleLogger = CustomerLogging.getConsoleLogger();
```

Comment:
There won't be any console logging on the client side anymore as it is up to the user to configure what goes to console and what not. We won't be enforcing anything. That's why specific console loggers were removed from all client side classes.